src/main/java/org/embulk/filter/unnest/UnnestFilterPlugin.java

```diff
@@ -0,0 +1,32 @@
+package org.embulk.filter.unnest;
+
+import org.embulk.config.Config;
+import org.embulk.config.ConfigSource;
+import org.embulk.config.Task;
+import org.embulk.config.TaskSource;
+import org.embulk.spi.FilterPlugin;
+import org.embulk.spi.PageOutput;
+import org.embulk.spi.Schema;
+
+public class UnnestFilterPlugin implements FilterPlugin {
+    public interface PluginTask extends Task {
+        @Config("json_column_name")
+        String getJsonColumnName();
+    }
+
+    @Override
+    public void transaction(ConfigSource config, Schema inputSchema, FilterPlugin.Control control) {
+        PluginTask task = config.loadConfig(PluginTask.class);
+
```

Comment:
Throw `ConfigException` here when the `json` column identified by `"json_column_name"` option does not exist. This is because detecting unrecoverable errors before InputPlugin and OutputPlugin start running reduces unnecessary access to the input and output data sources.